.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
need a comment explaining what this file is about, maybe with a link to the reference doc

Reply:
will add

Reply:
the comment is talking about direnv, but the code is talking about a flake ...